### ISSUE TITLE
Stable node_io release?

### ISSUE THREAD

**Issue by nex3** [pulyaevskiy/node-interop#57]
Are there plans to release a stable version of `node_io` soon? I'd like to use it, but I'm uncomfortable relying on a dev version.

**Comment by pulyaevskiy** [comment]
Good question!

I wanted to release stable version but I think it was a mistake from my end to start with 1.0. I’d like to reserve this version for when all functionality of dart:io is implemented (or at least the part which can be implemented with just built in Node modules).

I’ll take a look over the weekend at what’s left. 

Hmm.. I guess adding functionality is not a breaking change so can be done incrementally. In this case nothing really stops me from publishing stable version.

**Comment by pulyaevskiy** [comment]
Sorry for slowness here, I published `1.0.0` last night which completes implementation of file system objects and some more common classes. I also added missing docs and cleaned up public interface to make sure no implementation details are exposed.

There are still unimplemented parts (sockets, gzip codec,..) which I'll likely be tackling based on demand (or in a rare case when I have free time from other packages). So feel free to create an issue if there is something particular you're missing.

Thank you for your interest in this project!
